solidity/random-beacon/contracts/libraries/Authorization.sol

```diff
@@ -12,25 +12,588 @@
 //
```

Comment:
What do you think about naming it `BeaconAuthorization` and placing in `BeaconAuthorization.sol`? We already have `BeaconDkg.sol` and `BeaconInactivity.sol`.